nilearn/glm/tests/test_glm_reporter.py

```diff
@@ -585,3 +585,17 @@ def test_report_threshold_deprecation_warning(
         assert n_warnings == 1
     else:
         assert n_warnings == 0
+
+
+def test_report_make_glm_deprecation_warning(flm, contrasts):
+    """Test deprecation warning for nilearn.reporting.make_glm_report.
```

Comment:
Added this function in `glm` package instead of `reporting` to avoid adding a new test file and also because fixtures necessary to run the test are defined in this module.